app/src/main/res/layout/include_omnibar_toolbar.xml

```diff
@@ -115,6 +126,29 @@
 
                     </FrameLayout>
 
+                    <View
+                        android:id="@+id/cookieDummyView"
+                        android:layout_width="38dp"
```

Comment:
38dp is the same height as the `cookie_scene_1 FL.` How about making it an actual dimension?

Reply:
This makes sense to me. I see we have `38dp` also inside the `cookie_scene_2` layout. We can extract it to a dimension.